DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -47,6 +47,9 @@
             'miniAODDQM': ['DQMOfflineMiniAOD',
                            'PostDQMOfflineMiniAOD',
                            'DQMHarvestMiniAOD'],
+            'nanoAODDQM': ['DQMOfflineNanoAOD',
```

Comment:
Please make sure this sequence is covered by at least one run the matrix workflow.

There is a change to relval steps, but that change only changes the dataset...

Reply:
added for all relval involving NANOAOD